sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1672,7 +1672,9 @@ def fit(self, X, y, sample_weight=None):
         else:
             self.alphas_ = np.asarray(alphas[0])
 
-        # Refit the model with the parameters selected
+        # Refit the model with the parameters selected.
```

Comment:
The idea is to add documentation in the docstring and not some comments in the source code that is self-explanatory and that we will not be watched by our users.

Could you make sure to document in the docstring instead?